### ISSUE TITLE
Missing TypeScript declarations

### ISSUE THREAD

**Issue by panesofglass** [handsontable/react-handsontable#14]
I'm trying to create a working TypeScript declaration file, and I can't tell whether the issue I'm running into is related to how I'm attempting to mount the `HotTable` or the TypeScript d.ts file I've created. See #13.

The following seems to satisfy the TypeScript compiler. I'm happy to contribute it if it's helpful (and correct).

https://github.com/panesofglass/hottest/blob/11dc9da36ca10b5912bc203fe7a1201b4b625f16/src/react-handsontable.d.ts

That one doesn't compile correctly locally, but I found the following one appears to compile correctly, though the app still fails to run as described in #13:

https://github.com/panesofglass/hottest/blob/bd593d2e4cf3d7fc74cfb7aa252ae0a8e8ee5ca7/src/react-handsontable.d.ts

**Comment by panesofglass** [comment]
As I'm testing this further, it appears the Handsontable TypeScript declarations may be incorrect.

**Comment by aaronbeall** [comment]
Now that Handsontable has built-in type defs it would be great if react-handsontable did as well. I had a type def I made that was similar to the OP's version here, but it no longer works with the latest version `react-handsontable`. I just updated it to this and it seems to work:
```ts
declare module "react-handsontable" {
  import * as React from "react";
  import * as Handsontable from "handsontable";

  namespace HotTable {
    interface Props extends Handsontable._Handsontable.DefaultSettings {
      root?: string;
      settings?: Handsontable._Handsontable.DefaultSettings;
    }
  }
  class HotTable extends React.Component<HotTable.Props, {}> { }
  export = HotTable;
}
```
The `Handsontable._Handsontable.DefaultSettings` part is a bit weird.

**Comment by jansiegel** [comment]
The `1.1.0` version introduces the wrapper rewritten to TypeScript + and `index.d.ts` definition file in the `@handsontable/react` and `@handsontable-pro/react` packages.

Could you check if it helps in your cases, @panesofglass, @aaronbeall?

**Comment by wojciechczerniak** [comment]
Looks like the case is closed with #93. Please reopen or add new issue if we've missed something.
